languages/elixir/exercises/concept/freelance-rates/.docs/introduction.md

```diff
@@ -18,11 +18,11 @@ float = 3.45
 # => 3.45
 ```
 
-## Working with numbers
+### Working with numbers
 
 In the [`Integer`][integer-functions] and [`Float`][float-functions] modules you can find some useful functions for working with those types. Basic arithmetic operators are defined in the [`Kernel`][kernel-arithmetic-operators] module.
 
-## Conversion
+### Conversion
```

Comment:
I'm not entirely happy with putting this under `floating-point-numbers` because it's equally about integers and floats, but I have a hunch it's not allowed to have h2s that are not concepts (I'm waiting for a response)

Reply:
> @ErikSchierboom can the markdown parse being used understand YAML front matter? would that be less confusion than a second level heading with the concept slug?

Are you suggesting converting the entire document to YAML or to have YAML inside the Markdown?

> I'm not entirely happy with putting this under floating-point-numbers because it's equally about integers and floats, but I have a hunch it's not allowed to have h2s that are not concepts (I'm waiting for a response)

@iHiD Do we allow h2 headings that are not concepts? By the way, the easiest way around this is to introduce a new concept: `numeric-conversion` or something like that. As it applies to both integers and floating-point numbers, this is actually quite reasonable I feel.

> Hmm... I must have misunderstood. Will they will be replaced with the concept names that are in config.json? Because the slugs are ugly...

Yes, I think we'll replace them with the concept names. Right @iHiD?

Reply:
In my comment I have a link to an example how it is used with Jekyll. It's a yaml block at the top of the markdown which Jekyll uses for metadata about the markdown. 

I'm not suggesting changing the whole document to yaml

Reply:
Okay. Would replacing the slug with the concept names be okay? I'm not a big fan of the yaml block :)

Reply:
:shrug: I was just suggesting ideas that would avoid overloading the header markdown as a metadata marker.  I'm happy with your spec as is, whatever makes it easier for you all now.  It can always be changed later

Reply:
Indeed :)

Reply:
> Yes, I think we'll replace them with the concept names. Right @iHiD?

The file will be split up based on these headings, so they should match the concept slug. 

When the files are rendered the slugs will be replaced with the actual name, yes. The markdown files are already parsed and changes made, for context (e.g. for syntax highlighting). This is an extra part of that processing pipeline.

> Do we allow h2 headings that are not concepts? 

I'd rather not, no.

Reply:
But I could be persaded differently about the last bit, but would need to think about it more.

Reply:
I can live with only-concept-h2s-allowed.